test/handlers/route-api-test.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024, Pierre Bayerl
```

Comment:
You added yourself to CONTRIBUTORS; do you really want to keep the copyright to yourself (which you can if you really want), or else put "Compiler Explorer Authors" like in the other files?

Reply:
Thank you for this hint: I changed the text as you proposed (I originally just copied this copyright from some other file, where a name was provided).

Reply:
I hope it was okay that I added myself to CONTRIBUTORS (I read this somewhere in the docs or while creating the PR). My main motivation is to get the compression feature, but of course I am also very proud to be able to contribute to this great project :-) - even if it is only a small contribution!

Furthermore, I would like to know - once we agreed on this PR - how the roll-out process looks like: when can I expect that this new feature is available online (to update my slides :-) )?

Reply:
Absolutely! This should be rolled out in a few hours (I'll do it at lunchtime!)